drivers: gpio: Add B92 GPIO port F support & fix CI compilation errors

s07641069 opened the pull request:
- drivers: gpio: Add B92 GPIO port F support
- enable port E & F for B92
- fix CI compilation errors

andriy-bilynskyy commented:
The following west manifest projects have been modified in this Pull Request:

| Name | Old Revision | New Revision | Diff |
| ---- | ------------ | ------------ |------|
| hal_telink | https://github.com/telink-semi/hal_telink/commit/f59ca40b4e9d15ba8d6b7351a4a835ad28f3ded9 (telink_B92_adopt_ble) | https://github.com/telink-semi/hal_telink/commit/60ad4bf3940d6b8b9bb6c2ab933d931e9c90b74c (telink_B92) | [telink-semi/hal_telink@f59ca40b..60ad4bf3](https://github.com/telink-semi/hal_telink/compare/f59ca40b4e9d15ba8d6b7351a4a835ad28f3ded9..60ad4bf3940d6b8b9bb6c2ab933d931e9c90b74c) |

*Note: This message is automatically posted and updated by the Manifest GitHub Action.*

s07641069 committed:
drivers: gpio: Add B92 GPIO port F support

Enable port E & F for B92
Fix compliance checks & compilation errors after rebase
Update hal_telink module revision.

Signed-off-by: Alex Tsitsiura <s07641069@gmail.com>
